docs/main/Solutioning/openebs-on-kubernetes-platforms/gke.md

```diff
@@ -0,0 +1,299 @@
+---
+id: gke
+title: Replicated PV Mayastor Installation on Google Kubernetes Engine
+keywords:
+ - Replicated PV Mayastor Installation on Google Kubernetes Engine
+ - Replicated PV Mayastor Installation on GKE
+ - Replicated PV Mayastor - Platform Support
+ - Platform Support
+ - Google Kubernetes Engine
+ - GKE
+description: This section explains about the Platform Support for Replicated PV Mayastor.
+---
+# Replicated PV Mayastor Installation on Google Kubernetes Engine
+
+This document provides instructions for installing Replicated PV Mayastor on Google Kubernetes Engine (GKE).
+
+- GKE with local SSDs (Solid State Drive) are ephemeral because local SSDs are physically attached to the node’s host virtual machine instance, any data stored in them only exists on that node. Since the data stored on the disks is local, your application must be resilient to unavailable data.
+
+- A Pod that writes to a local SSD might lose access to the data stored on the disk if the Pod is rescheduled away from that node. Additionally, if the node is terminated, upgraded, or repaired the data will be erased.
+
+- Local SSDs cannot be added to an existing node pool.
+
+Using OpenEBS for GKE with Local SSDs offers several benefits, particularly in managing storage in a cloud-native way. 
+
+**Replication and Resilience:** OpenEBS can manage data replication across multiple nodes, enhancing data availability and resilience. Even though Local SSDs provide high performance, they are ephemeral by nature. OpenEBS can help mitigate the risk of data loss by replicating data to other nodes.
+
+**Performance:** Local SSDs provide high IOPS and low latency compared to other storage options. OpenEBS can leverage these performance characteristics for applications that require fast storage access.
+
+## GKE with Local SSD's
+
+1. GKE supports adding additional disks with local SSD.
```

Comment:
GKE supports adding additional disks with local SSD, while creating the cluster. Adding additional disks to existing node pool is not supported.